apps/desktop/desktop/src/main/resources/css/bisq_easy.css

```
@@ -1119,6 +1119,14 @@
     -fx-translate-y: -8;
 }
 
+.slider.max-or-fixed-amount-slider .thumb:hover,
+.slider.max-or-fixed-amount-slider .thumb:pressed {
+    -fx-background-color: -bisq-green-hover;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)